src/qml/controls/Theme.qml

```diff
@@ -0,0 +1,28 @@
+pragma Singleton
+import QtQuick 2.15
+
+QtObject {
+    property bool dark: false
+    property QtObject color: QtObject {
+        property color white: "#FFFFFF"
+        property color background: dark ? "black" : "white"
+        property color orange: dark ? "#F89B2A" : "#F7931A"
+        property color red: dark ? "#EC6363" : "#EB5757"
+        property color green: dark ? "#36B46B" : "#27AE60"
+        property color blue: dark ? "#2D9CDB" : "#3CA3DE"
+        property color purple: dark ? "#BB6BD9" : "#C075DC"
```

Comment:
I think, by mistake, you flipped the _dark toggle_ values for blue and purple as well, which were already correct.

**According to design scheme:**
![Screenshot from 2022-04-12 16-10-09](https://user-images.githubusercontent.com/85434418/162942592-5d8694aa-6706-4205-85a8-4cf44f085291.png)

**According to code:**
blue (when dark): `#2D9CDB`
purple (when dark): `#BB6BD9`

Reply:
thanks, addressed in https://github.com/bitcoin-core/gui-qml/commit/9b9d198407118475d6ba6dd315991f2a42c813b0